zwave-classifier.js

```diff
@@ -1249,6 +1334,32 @@ class ZWaveClassifier {
         };
       }
     }
+          
+    if (alarmProperty &&
+        zwValue &&
+        zwValue.class_id == COMMAND_CLASS.ALARM &&
+        zwValue.index == NOTIFICATION_CO_DETECTOR) {
+          const binarySensorValueId =
+          node.findValueId(COMMAND_CLASS.SENSOR_BINARY,
+            1,
+            SENSOR_BINARY_INDEX_SENSOR);
+          if (binarySensorValueId) {
+            this.addCarbonMonoxideProperty(node, binarySensorValueId);
+          }
+        }
+
+      if (alarmProperty &&
+          zwValue &&
+          zwValue.class_id == COMMAND_CLASS.ALARM &&
+          zwValue.index == NOTIFICATION_HEAT) {
+          const binarySensorValueId =
+          node.findValueId(COMMAND_CLASS.SENSOR_BINARY,
+            1,
+            SENSOR_BINARY_INDEX_SENSOR);
+          if (binarySensorValueId) {
+            this.addOverheatProperty(node, binarySensorValueId);
+          }
+        }
```

Comment:
The above 2 if's shouldn't be required. The CO and Overheat properties will be added due to the additions to the NOTIFICATION_SENSOR table.